### DISCUSSION TITLE
streamlink twitch gui authethification token

### DISCUSSION THREAD

**Discussion by Mixalisfr** [streamlink/streamlink#4245] [General]
helloo i have a small problem signing with my account when i click sign in  my browser(chrome redirects me to a bank page so i did a work around throught authenthification token thats dodnt seems to work eather i can watch streams but not with my account only as guest throught sreamlink twitch gui it tells me authethification failed try again

**Comment by bastimeyer** [comment]
Wrong repo. Locking...
